src/components/tooltip/Tooltip.tsx

```diff
@@ -90,6 +90,7 @@ export interface TooltipProps {
 
 type State = {
     isShown: boolean;
+    hasRendered: boolean;
```

Comment:
can you add a comment as to why this is needed for others to understand?

Reply:
ouch, just saw this. Will add a comment